src/test/java/org/mockitoutil/JUnitResultAssert.java

```diff
@@ -47,6 +50,17 @@ public JUnitResultAssert fails(int expectedFailureCount, Class expectedException
     }
 
     /**
+     * Expects single failure with specific exception and exception message.
+     * Automatically filters line numbers from exception messages.
+     */
+    public JUnitResultAssert fails(Class expectedException, String exceptionMessage) {
+        fails(1, expectedException);
+        Failure f = result.getFailures().iterator().next();
```

Comment:
I remember that at some point some `Iterator` implementations failed if `hasNext` was not invoked. I'd be careful on that one. (Better make a utility method in `org.mockito.internal.util.collections.Iterables`)